test/test_util.py

```diff
@@ -115,7 +115,7 @@ def popen_fail(*args, **kwargs):
         with self.assertRaises(subprocess.CalledProcessError) as exc_context:
             util.command_output([b"taga", b"\xc3\xa9"])
         self.assertEqual(exc_context.exception.returncode, 1)
-        self.assertEqual(exc_context.exception.cmd, b"taga \xc3\xa9")
+        self.assertEqual(exc_context.exception.cmd, 'taga \xc3\xa9')
```

Comment:
Here's why:
```
env LANG="en_US" python3
Python 3.4.5 (default, Jul  2 2016, 04:09:35) 
[GCC 6.1.1 20160621 (Red Hat 6.1.1-3)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> print('taga \xc3\xa9')
taga é
```

```
env LANG="en_US.utf-8" python3
Python 3.4.5 (default, Jul  2 2016, 04:09:35) 
[GCC 6.1.1 20160621 (Red Hat 6.1.1-3)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> print('taga \xc3\xa9')
taga Ã©
```
since the default windows cmd codepage on appveyor is mostly equivalent to en_US.iso-8859-1 (aka en_US), you can see why it worked there.